release/docker/build-image-multi-arch.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+
+set -e
+
+# Import libs
+. ../../lib/shell/file_management.sh
```

Comment:
Can you use absolute path here? If the script is run from different folder this can cause issue

Reply:
Then user should run from the default folder.
Because if they run from other folder not only this line will fail the cp line and other download line will also fail.
I dont want to tie this script to the git command to find out the repo root.

Reply:
How is that different from all other scripts we use? Also inorder to use this script they have to clone, fork the repo anyway. The thing is it can be easily missed and any change directory later can cause the file not found exception.

Reply:
@camerski would love your thoughts on this, I remember you mention it is ok to state that the script can only be run within a specific directory. If that is the case I will add more instructions in README about it.

Reply:
Ideally the script should work when executed from any working directory. `realpath $0` gives you the absolute location of the script, so you can use that to navigate to the `file_management.sh` dependency.

Basically, it's better to be able to run `/path/to/script -o tarball.tgz` than requiring `script -o /anotherpath/to/tarball.tgz`, because I'm more likely to be working in /anotherpath.

In the end it doesn't matter too much, though, just so long as the script fails gracefully if it can't source `file_management.sh`. No silent failures!

Reply:
The script does not fail silently it will complain the file not found, which is good for an early failure. I will update the readme to ask people run the script within specific folder.